pom.xml

```diff
@@ -7,7 +7,7 @@
   </parent>
 
   <artifactId>codedeploy</artifactId>
-  <version>1.17-SNAPSHOT</version>
```

Comment:
We do not adjust the version suffix. SNAPSHOT is for prereleases as we aggregate additional functionality, and then the automated plugin release process will handle removing this and pushing the untagged version to the appropriate release branch.